pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -7255,6 +7255,8 @@
 
   language_label: "Language"
 
+  careers_page_code_pride_alt_text: "Two female-presenting employees smile while holding the ends of a Code.org-branded rainbow banner. Behind them, several people are wearing Code.org shirts and waving rainbow flags while walking alongside a multi-colored Volkswagon bug."
```

Comment:
I really like that you added this description!

Nit: I think `female` has a biological connotation that I don't think you're intending. Maybe `feminine-presenting` or `woman-presenting`? Language for gender is evolving which makes this hard!